chat-demo-android/src/main/kotlin/com/twilio/chat/demo/services/FCMInstanceIDService.kt

```diff
@@ -1,16 +1,19 @@
 package com.twilio.chat.demo.services
 
-import com.google.firebase.iid.FirebaseInstanceIdService
-import org.jetbrains.anko.*
+import com.google.firebase.messaging.FirebaseMessagingService
+import org.jetbrains.anko.AnkoLogger
+import org.jetbrains.anko.debug
+import org.jetbrains.anko.startService
+
+class FCMInstanceIDService : FirebaseMessagingService(), AnkoLogger {
 
-class FCMInstanceIDService : FirebaseInstanceIdService(), AnkoLogger {
```

Comment:
Firebase update code, not related to crashlytics, split off.

Reply:
done